### PR TITLE
add config file ocp4.5ga minus1

### PR DESCRIPTION
**Pull request by AviadP**
Signed-off-by: aviadp <apolak@redhat.com>

### CODE REVIEW

File: conf/ocp_version/ocp-4.5-ga-minus1-config.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  client_version: '4.5-ga'
+
+DEPLOYMENT:
+  ocp_url_template: "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/{version}/{file_name}-{os_type}-{version}.tar.gz"
+  installer_version: "4.5-ga"
+  terraform_version: "0.12.26"
+  ocp_channel: "stable"
+  ocp_version_index: -2
+ENV_DATA:
+  vm_template: 'rhcos-4.3.0-x86_64-vmware'
```
Comment (petr-balogh) [inline conf/ocp_version/ocp-4.5-ga-minus1-config.yaml:14]: This template doesn't sound good to me for 4.5

Comment (AviadP) [reply conf/ocp_version/ocp-4.5-ga-minus1-config.yaml:14]: ye missed it

**Review by petr-balogh — APPROVED** [review]

**Comment by AviadP** [comment]
verified by this build: https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/11744/parameters/

**Review by ebenahar — APPROVED** [review]

**Comment by ebenahar** [comment]
@AviadP can you please add a description why this is needed?

**Pull request merged**